batch/batch/exceptions.py

```diff
@@ -5,15 +5,23 @@ def __init__(self, message, severity):
         self.message = message
         self.ui_error_type = severity
 
-    def api_response(self):
+    def http_response(self):
```

Comment:
ah, I kinda saw the purpose as "format this error as an HTTP response", so I updated the name to match. (in my mind the  UI version handles the error somewhat separately from the HTTP response; it sets a message field in the response page, but otherwise the response page is the same)

I could see it both ways, since the use case for "format this error as an HTTP response" is predominantly (entirely?) within the API, and wouldn't be upset if it changed, but probably won't change it for this PR.